src/attributes.jl

```diff
@@ -218,6 +230,64 @@ A `Bool` indicating if the solver interprets certain quadratic constraints as se
 """
 struct SupportsConicThroughQuadratic <: AbstractSolverAttribute end
 
+## Instance attributes
+
+"""
+    ObjectiveSense()
+
+The sense of the objective function, an `OptimizationSense` with value `MinSense`, `MaxSense`, or `FeasiblitySense`.
+"""
+struct ObjectiveSense <: AbstractInstanceAttribute end
+
+@enum OptimizationSense MinSense MaxSense FeasibilitySense
+
+"""
+    NumberOfVariables()
+
+The number of variables in the instance.
+"""
+struct NumberOfVariables <: AbstractInstanceAttribute end
+
+"""
+    ListOfVariableReferences()
+
+A `Vector{VariableReference}` with references to all variables present
+in the instance (i.e., of length equal to the value of `NumberOfVariables()`).
+"""
+struct ListOfVariableReferences <: AbstractInstanceAttribute end
+
+"""
+    ListOfConstraintReferences{F,S}()
+
+A `Vector{ConstraintReferences{F,S}}` with references to all constraints of
+type `F`-in`S` in the instance (i.e., of length equal to the value of `NumberOfConstraints{F,S}()`).
+"""
+struct ListOfConstraintReferences{F,S} <: AbstractSolverInstanceAttribute end
```

Comment:
Shouldn't this be a `AbstractInstanceAttribute` ? You have added `Solver`

Reply:
Good catch